.github/workflows/ci.yml

```diff
@@ -111,21 +115,12 @@ jobs:
 
             git checkout $cassette_branch
 
-            if git merge --no-commit --no-ff ${{ github.event.pull_request.base.ref }}; then
-              echo "Using cassettes from mirror branch, synced to upstream branch '${{ github.event.pull_request.base.ref }}'"
-            else
-              echo "Could not merge upstream changes to cassettes. Using cassettes from ${{ github.event.pull_request.base.ref }}."
-              git merge --abort
-              git checkout ${{ github.event.pull_request.base.ref }}
-
-              # Delete branch to prevent conflict when re-creating it
-              git branch -D $cassette_branch
-            fi
-            echo "cassette_branch=$(git branch --show-current)" >> $GITHUB_OUTPUT
+            git merge --no-commit --strategy-option ours origin/${{ github.event.pull_request.base.ref }}
```

Comment:
merge with --ours strategy: we pick what can be picked from master, otherwise we just use the cache built previously.

Reply:
By the way, why the `--no-commit`? I see it was there before but the reason isn't obvious to me.